Majasdarbs/03-majasdarbs/Gvido-CV-CSS/responsive.css

```diff
@@ -0,0 +1,10 @@
+@media screen and (max-width: 650px) {
```

Comment:
Check common media breakpoints https://www.freecodecamp.org/news/css-media-queries-breakpoints-media-types-standard-resolutions-and-more/